docs/data/base/components/accordion/accordion.md

```diff
@@ -0,0 +1,15 @@
+---
+product: base
+title: React Accordion component
+githubLabel: 'component: accordion'
+waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/accordion/
+---
+
+# Accordion
+
+<p class="description">Accordions let users show and hide sections of related content on a page.</p>
+
+:::info
+💡 The Base UI Accordion component is still in development.
+If you're in need of it, please give a shout about your need of it in [**this issue**](https://github.com/mui/material-ui/issues/27170) to help us prioritize the next batch of new components.
```

Comment:
> If you're in **need of it**, please give a shout about your **need of it**

That's a bit repetitive, could you somehow rephrase it?

Reply:
Let me tag @samuelsycamore for the copywriting magic over here 😬

Reply:
How about:

> If you're in need of it, please give us a shout about it in this issue to help us prioritize the next batch of new components.

What do y'all think?

Reply:
For something like this I'd keep it short and direct:

```suggestion
Help us prioritize the next batch of new components by sharing your needs in [this GitHub issue](https://github.com/mui/material-ui/issues/27170).
```

Reply:
Sweet — much better 👌

Reply:
The text should also be updated in other components.

Reply:
🤦 Done ⎯ thanks for pointing out!